protocol/x/affiliates/types/constants.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+var (
+	DefaultAffiliateTiers = AffiliateTiers{
+		Tiers: []AffiliateTiers_Tier{
+			{
+				ReqReferredVolumeQuoteQuantums: 0,
```

Comment:
Let's add human readable comment on the values (e.g. 1 million USDC, 10%) since these values are important